scripts/gulp/create-bundle.js

```diff
@@ -117,6 +118,12 @@ module.exports = function createBundle(config, buildOpts) {
 
   var bundle = browserify([], bundleOpts);
 
+  bundle.transform(stringify, {
```

Comment:
Given that this transform is applied unconditionally, it probably should. I'll give it a go.